lib/commands/device/start-service.ts

```diff
@@ -0,0 +1,144 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Flags, Args } from '@oclif/core';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, getCliUx, stripIndent } from '../../utils/lazy';
+import type { BalenaSDK } from 'balena-sdk';
+
+export default class DeviceStartServiceCmd extends Command {
+	public static description = stripIndent`
+		Start containers on a device.
+
+		Start containers on a device.
+		The --service flag should be provided to start services' containers.
+
+		Multiple devices and services may be specified with a comma-separated list
+		of values (no spaces).
+		`;
+	public static examples = [
+		'$ balena device start-service 23c73a1 --service myService',
+		'$ balena device start-service 23c73a1 -s myService1,myService2',
+	];
+
+	public static args = {
+		uuid: Args.string({
+			description: 'comma-separated list (no blank spaces) of device UUIDs',
+			required: true,
+		}),
+	};
+
+	public static usage = 'device start-service <uuid>';
+
+	public static flags = {
+		service: Flags.string({
+			description: 'comma-separated list (no blank spaces) of service names',
+			char: 's',
+		}),
```

Comment:
Should we actually mark the service flag as required (both here in in the stop command), given that we do not support this command without it afaict?

```suggestion
		service: Flags.string({
			description: 'comma-separated list (no blank spaces) of service names',
			char: 's',
			required: true,
		}),
```
and then we can assume that `serviceNames` & `options.service` can't be undefined.

Reply:
good suggestion. have made the changes.